src/pages/ShareCodePage.js

```diff
@@ -41,7 +41,7 @@ class ShareCodePage extends React.Component {
         const isReport = this.props.report != null && this.props.report.reportID != null;
         const subtitle = ReportUtils.getChatRoomSubtitle(this.props.report);
 
-        const url = isReport ? `${CONST.NEW_EXPENSIFY_URL}r/${this.props.report.reportID}` : `${CONST.NEW_EXPENSIFY_URL}details?login=${encodeURIComponent(this.props.session.email)}`;
+        const url = isReport ? `${CONST.NEW_EXPENSIFY_URL}r/${this.props.report.reportID}` : `${CONST.NEW_EXPENSIFY_URL}a/${this.props.session.accountID}`;
```

Comment:
Can we use [getReportRoute](https://github.com/Expensify/App/blob/1af376d64b69aae8690619e5d2e0f489c185ed16/src/ROUTES.js#L68) and [getProfileRoute](https://github.com/Expensify/App/blob/1af376d64b69aae8690619e5d2e0f489c185ed16/src/ROUTES.js#L117) instead?